account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -0,0 +1,374 @@
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import ormcache
+from odoo.tools.safe_eval import safe_eval
+
+
+class AccountMulticompanyEasyCreationWiz(models.TransientModel):
+    _name = "account.multicompany.easy.creation.wiz"
+    _description = "Wizard Account Multi-company Easy Creation"
+
+    def _default_sequence_ids(self):
+        exclude_seq_list = self.env["ir.config_parameter"].get_param(
+            "account_multicompany_easy_creation.exclude_sequence_list",
+            [
+                False,
+                "aeat.sequence.type",
```

Comment:
Is this appropriate for a module outside of l10n-spain?

Reply:
As both are OCA modules, this is allowed as "trick" for avoiding glue modules.

Reply:
OK, but I think there should be some comment here explaining that.